go.mod

```diff
@@ -17,12 +17,12 @@ require (
 	github.com/coredns/coredns v1.1.2
 	github.com/digitalocean/godo v1.10.0 // indirect
 	github.com/docker/go-connections v0.3.0
-	github.com/elazarl/go-bindata-assetfs v0.0.0-20160803192304-e1a2a7ec64b0
```

Comment:
Were all of these updates strictly necessary for the PR?

Reply:
I meant to comment on this. I don't think they were, but in order to actually rebuild the protobuf I had to update my local tools with `make tools` and that did this. I'm not really sure what the correct way to keep all our tools up to date without this is but open to suggestions.

It would be fiddly to have to go and manually remove the changes `make tools` made while keeping the legit ones I need in this PR but I can - I feel like we should just fix `make tools` though if this is a problem? Thoughts welcome.

Reply:
Yeah `make tools` shouldn't monkey with this at all. I had to fix the circle CI build to restore `go.{mod,sum}` at some points because of this, but didn't think to go back and figure out how to avoid the original problem. Please try reverting the `go.{mod,sum}` changes while I see about fixing the root.

Reply:
Oh I also thought this file was a mix of legit dep changes and tools stuff but actually in the end after a tidy it wasn't so I'll just revert this as you said. Tool fix would be great though.

Reply:
https://github.com/hashicorp/consul/pull/6738

Reply:
Reverted!